DATA/UNIVERSE/SYSTEMS/Br04/Br04.ini

```diff
@@ -4555,12 +4884,79 @@ archetype = space_industrial01
 reputation = br_m_grp
 parent = Br04_02
 
+[Object]

```

Comment:
Very odd this one and the object above come so far down in the file. Would be more fitting to have them near the entries they belong to.